.storybook/preview.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Global,
+  ThemeProvider,
+  themes,
+  createReset,
+  convert,
+  styled,
+  useTheme,
+} from "@storybook/theming";
+import type { Preview } from "@storybook/react";
+import { initialize, mswDecorator } from "msw-storybook-addon";
+import React from "react";
+
+// Initialize MSW
+initialize({
+  onUnhandledRequest(req) {
+    if (req.url.origin !== document.location.origin) {
+      console.error(
+        `[MSW] %s %s %s (UNHANDLED)`,
+        new Date().toTimeString().slice(0, 8),
+        req.method.toUpperCase(),
+        req.url.href
+      );
+    }
+  },
+});
+
+const Panels = styled.div<{ orientation: "right" | "bottom" }>(
+  ({ orientation }) => ({
+    flexDirection: orientation === "right" ? "row" : "column",
+  }),
+  {
+    width: "100vw",
+    height: "100vh",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    gap: 20,
+  }
+);
+
+const Panel = styled.div<{ orientation: "right" | "bottom" }>(
+  ({ orientation }) => ({
+    width: orientation === "right" ? 420 : 900,
+    height: orientation === "right" ? 640 : 300,
+    overflow: "auto",
+  }),
+  ({ theme }) => ({
+    position: "relative",
+    outline: `1px solid ${theme.color.border}`,
+    background: theme.background.content,
+    color: theme.color.defaultText,
+    fontSize: theme.typography.size.s2 - 1,
+  })
+);
+
+const ThemedSetRoot = () => {
+  const theme = useTheme();
+  React.useEffect(() => {
+    document.body.style.background = theme.background.content;
+    document.body.style.color = theme.color.defaultText;
+    document.body.style.fontSize = `${theme.typography.size.s2 - 1}px`;
+  });
+  return null;
+};
+
+export const decorators = [
+  // Provide the MSW addon decorator globally
+  mswDecorator,
+
+  // Render two panels side-by-side or stacked, depending on selected orientation
+  // Note this assumes any play functions are equipped to deal with multiple canvases
+  (StoryFn, { globals, parameters }) => {
+    const theme = globals.theme || parameters.theme || "right";
```

Comment:
<img width="1136" alt="Screen Shot 2023-06-15 at 11 26 06 AM" src="https://github.com/chromaui/addon-visual-tests/assets/5882088/9fb3b279-9f1e-4f51-ae51-28844b9a852f">
Also, question about the themes. It looks like the background in the components doesn't change, and the font is lighter. Is this what you expected? Do future updates add the other stylings one would expect?

Reply:
`theme` in this context is the value of `globalTypes.theme` as configured at the bottom of `preview.tsx`, and it's always a string ("light", "dark", "right" or "bottom"). 

`globals.theme` is the value of the toolbar selection (if any)
`parameters.theme` is a potential story-level value, but can be overruled using the toolbar
We default to "right" because that's what the addon was designed for primarily.

I suspect you may be confused about `theme` in the context of Emotion, which is an object originating from `@storybook/theming`.